### PR TITLE
Add signals to the Profile and Resolution interfaces

### LINKED ISSUE #7: The Profile interface needs a signal for active profile changes

### PR DESCRIPTION
**Pull request by Hjdskes**
Currently this only holds the commit adding a signal for active profile changes to the Profile interface, hence its WIP status. I'll add the other commits as I finish them.

Closes #7, #8.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,8 @@ The **org.freedesktop.ratbag1.Profile** interface provides:
 - Methods:
   - SetActive(void) -> set this profile to be the active profile
   - GetResolutionByIndex(uint) -> returns the object path for the given index
+- Signals:
+  - ActiveProfileChanged -> active profile changed, carries index of the currently active profile
```
Comment (Hjdskes) [inline README.md]: We might want to change the wording here. "Currently active profile" may also be interpreted as the profile that's active when the active profile change occurs. Thoughts?

Same applies for the signal description of the active resolution.

**Comment by Hjdskes** [comment]
#8 also mentions a default resolution, but there is no setter for that. What do I do with that @whot? Other than that, I think this is done so I'll mark it as such.

File: README.md (hunk 1 above)
Comment (whot) [reply README.md]: yeah, just change it to "new active profile", that removes the ambiguity

Comment (Hjdskes) [reply README.md]: Done

**Review by whot — APPROVED** [review]

**Comment by whot** [comment]
regarding the default resolution: there's a setter for that one in libratbag, looks like ratbagd is just missing that code. should be an easy fix if you have too much freetime ;)

**Comment by Hjdskes** [comment]
On it!

**Pull request merged**